how-to/onboarding-guide.md

```diff
@@ -26,37 +26,75 @@ Chief. In the future we hope to find someone with interest in leading peer revie
 
 ## Where to find new editors
 
-As we build our pyOpenSci community our pool of potential editors will grow.
-A few options to consider include:
+The pyOpenSci community continues to grow rapidly, and as such we now have a much stronger reach on social media than ever before. While you will collaborate closely with the pyOpenSci Community Manager to recruit editors through social media campaigns, also consider the following sources:
+
+The pyOpenSci community continues to grow rapidly, and as such we now have a much stronger reach on social media than ever before. While you will collaborate closely with the pyOpenSci Community Manager to recruit editors through social media campaigns, also consider the following sources:
+
 * Contributors who have reviewed for pyOpenSci
 * Contributors who have submitted a package to pyOpenSci
 * Contributors who have served as a guest editor
 * Colleagues that you know who have reviewed for JOSS or rOpenSci who have Python expertise
 
+pyOpenSci maintains a database of contributors, and the Community Manager can assist you in identifying individuals who may be a good fit. Please reach out to them either in a private Slack message or via email, at [media@pyopensci.org](mailto:media@pyopensci.org).
+
+
+pyOpenSci maintains a database of contributors, and the Community Manager can assist you in identifying individuals who may be a good fit. Please reach out to them either in a private Slack message or via email, at [media@pyopensci.org](mailto:media@pyopensci.org).
```

Comment:
i am not sure how i missed this when i reviewed - i must have been tired. many paragraphs are duplicated.

Reply:
I don't know how I missed it, either! I have a hunch that something's pretty borked on my repo...

Reply:
what is weird is that i also missed that this isn't a pr from your fork. This is a pr from a branch in the pyos repo.

i normally catch that but i obviously didn't. The commits are also really odd. i am not sure what happened here!

<img width="896" alt="Screenshot 2024-06-13 at 5 32 03 PM" src="https://github.com/pyOpenSci/software-peer-review/assets/7649194/1d98f6ee-ef17-47c8-8691-e06d9d0fe042">

perhaps we can walk through pr's from your fork together in our next checkin to see how things are getting setup and to troubleshoot together?

Reply:
see this new pr i opened here #290  - you can see in the image it's from my fork. whereas this pr is not.  🤷🏻‍♀️ 

<img width="1074" alt="Screenshot 2024-06-13 at 5 35 02 PM" src="https://github.com/pyOpenSci/software-peer-review/assets/7649194/3020c6f0-7d72-4482-a34c-f5754226565f">